packages/suite-desktop-core/e2e/support/fixtures.ts

```diff
@@ -47,10 +45,6 @@ const test = base.extend<Fixtures>({
         const suiteGuidePage = new SuiteGuide(window);
         await use(suiteGuidePage);
     },
-    topBar: async ({ window }, use) => {
```

Comment:
This page object didn't offere anything specific and was dealing with very small area of product.
I moved the navigation to the corresponding fixtures
It is much easier to just import Dashboard and the do dashaboardPage.navigateTo()